things/thing-18.md

```diff
@@ -0,0 +1,177 @@
+---
+HandheldFriendly: true
+author: ANDS
+description: |
+    The Australian National Data Service (ANDS) is a program funded by the
+    Australian Government to develop research data infrastructure and enable
+    more effective use of Australia's research data assets.
+keywords: 'data, research, research data, australia'
+og:description: |
+    The Australian National Data Service (ANDS) is a program funded by the
+    Australian Government to develop research data infrastructure and enable
+    more effective use of Australia's research data assets.
+og:site_name: ANDS
+og:title: 'Thing 18: Data interviews: talk the talk'
+og:type: website
+og:url: 'https://www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-18'
+robots: 'index,follow'
+title: 'Thing 18: Data interviews: talk the talk - ANDS'
+viewport: |
+    width=device-width, initial-scale=1.0, maximum-scale=1.0,
+    user-scalable=0
+---
+
+# Thing 18: Data interviews: talk the talk
+**[&lt;&lt;Jump to Thing 17](thing-17.md) &nbsp; &nbsp; [Jump to Thing 19&gt;&gt;](thing-19.md)**
+
+Learn some tips and tricks for responding to data queries and starting a
+data conversation.
+
+-   **Getting started**: asking the essential questions to create
+    metadata records, or find out about research data needs
+-   **Learn more**: conversations starters about research data services,
+    or think about how interviews uncover vital information useful for
+    data planning
+-   **Challenge me**: in depth interviews reveal the extent to which
+    data reuse is dependent on tools and software - what’s your advice?
+
+* [Getting started](#getting-started)
+* [Learn more](#learn-more)
+* [Challenge me](#challenge-me)
+
+## Getting started 
+### Starting a data conversation
+
+Thing 18 offers a simple way to think about key messages and possible
+responses that can be used to quickly and succinctly pitch research data
+management to researchers. The responses can be contextualised to suit
+what is available at your organisation.
+
+1.  Start by scrolling to the very last link on the bottom of [this
+    page](https://www.ands.org.au/working-with-data/data-management/overview/data-management-skills/information-specialists-and-data-librarian-skills)
+    and either open the PDF or download the Word version of the
+    "**What’s my pitch?**” document.
+2.  If you can, fill in the gaps in the document with your institutional
+    services. If you don’t know the answer to some, see if you can find
+    out. The result will be your own version of the toolkit.
+
+**Consider:** what are some ideas you may have for conversation
+starters?
+
+Do you have a question?  Want to share a resource?
+- Post to the [Data Librarians Slack group](https://tiny.cc/data-librarians) to connect with the community.
+- Tweet to [@ardc_au](https://twitter.com/ARDC_AU) using hashtag #23things
+
+Go to [Thing 19](thing-19.md "thing 19")
+Exploring APIs andApps or to [All Things](index.md)
```

Comment:
missing space in "and Apps" here and throughout

Reply:
Fixed each occurrence